app/src/lib/stores/app-store.ts

```diff
@@ -1624,36 +1613,22 @@ export class AppStore extends TypedBaseStore<IAppState> {
   }
 
   private startPullRequestUpdater(repository: Repository) {
-    if (this.currentPullRequestUpdater) {
-      this.stopPullRequestUpdater()
-    }
-
     // We don't want to run the pull request updater when the app is in
```

Comment:
Suggested change for the comment above about maintaining existing behavior.

```suggestion
    this.stopPullRequestUpdater()

    // We don't want to run the pull request updater when the app is in
```